src/tcgwars/logic/impl/gen7/ShinyVault.groovy

```diff
@@ -0,0 +1,398 @@
+package tcgwars.logic.impl.gen;
+
+import tcgwars.logic.impl.gen6.Breakthrough;
+
+import tcgwars.logic.impl.gen7.CelestialStorm;
+import tcgwars.logic.impl.gen7.DragonMajesty;
+import tcgwars.logic.impl.gen7.ForbiddenLight;
+import tcgwars.logic.impl.gen7.BurningShadows;
+import tcgwars.logic.impl.gen7.SunMoon;
+import tcgwars.logic.impl.gen7.GuardiansRising;
+import tcgwars.logic.impl.gen7.UltraPrism;
+import tcgwars.logic.impl.gen7.ShiningLegends;
+import tcgwars.logic.impl.gen7.CrimsonInvasion;
+import tcgwars.logic.impl.gen7.TeamUp;
+import tcgwars.logic.impl.gen7.LostThunder;
+import tcgwars.logic.impl.gen7.SunMoonPromos;
+
+import static tcgwars.logic.card.HP.*;
+import static tcgwars.logic.card.Type.*;
+import static tcgwars.logic.card.CardType.*;
+import static tcgwars.logic.groovy.TcgBuilders.*;
+import static tcgwars.logic.groovy.TcgStatics.*
+import static tcgwars.logic.effect.ability.Ability.ActivationReason.*
+import static tcgwars.logic.effect.EffectType.*;
+import static tcgwars.logic.effect.Source.*;
+import static tcgwars.logic.effect.EffectPriority.*
+import static tcgwars.logic.effect.special.SpecialConditionType.*
+import static tcgwars.logic.card.Resistance.ResistanceType.*
+
+import java.util.*;
+import org.apache.commons.lang.WordUtils;
+import tcgwars.entity.*;
+import tcgwars.logic.*;
+import tcgwars.logic.card.*;
+import tcgwars.logic.card.energy.*;
+import tcgwars.logic.card.pokemon.*;
+import tcgwars.logic.card.trainer.*;
+import tcgwars.logic.effect.*;
+import tcgwars.logic.effect.ability.*;
+import tcgwars.logic.effect.ability.Ability.*;
+import tcgwars.logic.effect.advanced.*;
+import tcgwars.logic.effect.basic.*;
+import tcgwars.logic.effect.blocking.*;
+import tcgwars.logic.effect.event.*;
+import tcgwars.logic.effect.getter.*;
+import tcgwars.logic.effect.special.*;
+import tcgwars.logic.util.*;
+
+/**
+ * @author axpendix@hotmail.com
+ */
+public enum ShinyVault implements CardInfo {
```

Comment:
`CardInfo` is now replaced with `LogicCardInfo`

I should update carddb-tools sometime soon to automatically do this